ceph/ceph_admin/dashboard.py

```diff
@@ -112,3 +113,39 @@ def validate_enable_dashboard(cls, username, password):
         LOG.info(
             "Dashboard API login is successful, status code : %s" % resp.status_code
         )
+
+
+def enable_monitoring(cls):
+    """Method to enable the monitoring stack module.
+
+    if user bootstrap with skip-dashboard option
+    then enabling the monitoring stack module.
+
+    Args:
+        cls (CephAdmin object) : cephadm instance object.
+    """
+    out, _ = cls.shell(args=["ceph", "mgr", "services"])
+    formatted = json.loads(out)
+
+    # Getting the ip address to set monitoring stack URL's
+    url = formatted.get("prometheus")
+
+    # Finding IP address using IPV4 format
+    pattern = re.compile(r"(\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3})")
+    ip = pattern.findall(url)[0]
+
+    # To enable monitoring stack
+
+    MONITORING_ENABLE_COMMANDS = [
+        f"ceph dashboard set-alertmanager-api-host 'http://{ip}:9093'",
+        "ceph dashboard set-alertmanager-api-ssl-verify False",
+        f"ceph dashboard set-prometheus-api-host 'http://{ip}:9095'",
+        "ceph dashboard set-prometheus-api-ssl-verify False",
+        f"ceph dashboard set-grafana-api-url 'https://{ip}:3000'",
+        "ceph dashboard set-grafana-api-ssl-verify False",
+    ]
+
+    for cmd in MONITORING_ENABLE_COMMANDS:
+        out, err = cls.shell(args=[cmd])
+        LOG.info("STDOUT:\n %s" % out)
+        LOG.error("STDERR:\n %s" % err)
```

Comment:
can we also verify the endpoint using API calls?

Reply:
Yes thanks @sunilkumarn417 for the review ,
I will try to add a validation part for the monitoring service as well